.dockerignore

```diff
@@ -0,0 +1 @@
+node_modules/
```

Comment:
How about ignoring every package/module's `node_modules` instead of just the root one?
```suggestion
**/node_modules/
```